pkgs/games/unvanquished/default.nix

```
@@ -0,0 +1,139 @@
+{ lib, stdenv, writeScript, fetchzip, fetchurl, fetchgit, buildFHSUserEnv, runCommandNoCC, makeDesktopItem
+, gcc, cmake, gmp, libGL, zlib, ncurses , geoip, lua5, nettle
+, curl, SDL2, freetype, glew , openal, libopus, opusfile, libogg
+, libvorbis, libjpeg, libwebp, libpng
+, libtheora # TODO: drop once we get to 0.52
+, cacert, aria2 # to download assets
+}:
+
+let version = "0.51.2";
+    binary-deps-version = "4";
```

```suggestion
let
  version = "0.51.2";
  binary-deps-version = "4";
```

I think we you need to reflow the entire file. I would suggest using a formatter like nixpkgs-fmt.